src/base/utils/net.cpp

```diff
@@ -0,0 +1,13 @@
+#include "net.h"
+#include <QHostAddress>
+
+namespace Utils
+{
+    namespace Net
+    {
+        bool isValidIP(const QString &ip)
+        {
+            return !QHostAddress(ip).isNull();
```

Comment:
I had a plan to implement additional filtering in `Session::setBannedIPs()` and this function `isValidIP()` shoud be useful there. It was a cause to create net.cpp and move only one function there. And at that time `isValidIP()` was more complicated. It happened later to became so simple, thanks to __evsh__ .